Uplift/Views/Cells/FacilitiesDropdownCollectionViewCell.swift

```diff
@@ -0,0 +1,185 @@
+//
+//  FacilitiesDropdownCollectionViewCell.swift
+//  
+//
+//  Created by Cameron Hamidi on 11/6/19.
+//
+
+import UIKit
+
+class FacilitiesDropdownCollectionViewCell: UICollectionViewCell {
+
+    private enum FacilityName: String {
+        case bowlingLanes = "BOWLING LANES"
+        case equipment = "EQUIPMENT"
+        case fitnessCenter = "FITNESS CENTER"
+        case gymnasium = "GYNMASIUM"
+        case swimmingPool = "SWIMING POOL"
+    }
+
+    private let headerImageView = UIImageView()
+    private let headerNameLabel = UILabel()
+    private let headerOpenLabel = UILabel()
+    private let headerViewHeight: CGFloat = 52
+    private let minimumInteritemSpacing: CGFloat = 16
+
+    private var collectionView: UICollectionView!
+    private var collectionViewHeight: CGFloat!
+    private var dropdownView: DropdownView!
+    private var facility: Facility!
+    private var headerView: UIView!
```

Comment:
make `private let`

Reply:
All of those values will change though, and many of them are changed in class functions (that can't be called if I try and set the value of the `private let` before calling the initializer).